src/docs/sphinx/dev_guide/updating_tpls.rst

```diff
@@ -172,105 +186,119 @@ Updating TPLs
+    located in ``axom/scripts/spack/radiuss-spack-configs``.
 
 There is no software installation process for these dependencies in the 
 traditional sense. To update one of these packages in Axom, simply go into
-its directory in Axom and check out a new version. If a version is intended
-to be changed in the Axom repo, make the version change on a branch and 
-submit a GitHub pull request as you would do for other software changes.
-More info on :ref:`building-axom-label`.
+the directory where the submodule lives in Axom and check out a new version.
+If a version is intended to be changed in the Axom repo, make the version change
+on a branch and submit a GitHub pull request as you would do for other software
+changes. More info on :ref:`building-axom-label`.
 
 Built-in TPLs
 -------------
 
 Axom uses several lightweight, header-only libraries internally, which are
-exposed for downstream customers to use if they wish.
+exposed for downstream customers to use if they wish. These are:
 
-  * `CLI11 <https://github.com/CLIUtils/CLI11>`_ is a command line parser
-    for C++11 and beyond that provides a rich feature set with a simple and
+  * `CLI11 <https://github.com/CLIUtils/CLI11>`_, a command line parser
+    for C++ and beyond that provides a rich feature set with a simple and
     intuitive interface.
-  * `fmt <https://github.com/fmtlib/fmt>`_ is an open-source formatting
+  * `fmt <https://github.com/fmtlib/fmt>`_, an open-source formatting
     library providing a fast and safe alternative to C stdio and C++ iostreams.
-  * `sol <https://github.com/ThePhD/sol2>`_ is a C++ library binding to Lua.
-  * `Sparsehash <https://github.com/sparsehash/sparsehash>`_ contains several
-    hash-map implementations.
+  * `sol <https://github.com/ThePhD/sol2>`_, a C++ library binding to Lua.
+  * `Sparsehash <https://github.com/sparsehash/sparsehash>`_, which contains
+    several hash-map implementations.
 
-.. note:: Axom patches all built-in TPLs to be under the ``axom`` namespace.
-   This is to prevent symbol collisions with other projects, either our
-   dependencies or downstream customers who wish their own versions.  For
+.. note:: Axom patches its built-in TPLs so that they reside in the ``axom`` namespace
+   which prevents symbol collisions with other projects, either our
+   dependencies or downstream customers who wish to use their own versions.  For
    example, ``fmt::format("foo")`` is ``axom::fmt::format("foo")``.
 
-They can be found in the directory: ``axom/src/thirdparty/axom``. The basic 
-instructions on how to update a built-in TPL are as follows:
+These TPLs are located in the directory: ``axom/src/thirdparty/axom``. The basic 
+instructions on how to update a built-in TPL are:
 
 #. Download the new release and override the source that is already there.
-   This can often involve removing files no-longer needed but most of the
-   current ones are a single header file.
+   This may involve removing files no-longer needed.
```

Comment:
```suggestion
   This may involve removing files no longer needed.
```
(minor suggestion for style and clarity)

Reply:
Fixed.